e2e/tests/profile-types-view/profile-types.spec.ts

```diff
@@ -80,4 +85,14 @@ test.describe('Profile types view', () => {
       await expect(exploreProfilesPage.getSceneBody()).toHaveScreenshot();
     });
   });
+
+  test('Settings button', async ({ exploreProfilesPage }) => {
```

Comment:
We chose the "Profile types" view because it's not the default view, so going back with the browser button should properly set it back as the active view.